src/code/views/menu-bar-view.coffee

```diff
@@ -8,6 +8,10 @@ module.exports = React.createClass
   displayName: 'MenuBar'
 
   componentWillMount: ->
+    # need to use direct DOM events because the event needs to be captured
+    if window.addEventListener
+      window.addEventListener 'click', @checkBlur, true
```

Comment:
This is clever. I like it. I have a couple of thoughts, however.
- Have you tested this with dragging? I would think that the click, and therefore the blur, would happen after the drag was complete. It's worth considering whether the handler should be on 'mousedown'.
- I believe the CFM menu is sometimes stickier than it should be and so a similar approach could be used there, although not necessarily part of this PR.